frontend/src/components/NewBoardModal.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from "react";
+import "./NewBoardModal.css";
+
+const ModalOptions = Object.freeze({
+  CATEGORY: "select-a-category",
```

Comment:
Looks like these options are used in multiple places. I mentioned in a file above. In such a case, it's better to define this object in a separate file and import in all the places it's used.